### PR TITLE
Outfit datum

### PR DESCRIPTION
**Pull request by AnturK**
Because there was no unified and readable way to do this.
- [x] Admin & ERT outfits
- [x] Job outfits
- [x] Custom outfits

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Does... does this make it possible in the future to make the holosuit disguise as entire kits?

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 1]
```diff
@@ -471,4 +471,8 @@
 		else
 			burn()
 
-	return shredded
\ No newline at end of file
+	return shredded
+
+/mob/living/carbon/human/proc/equipOutfit(outfit)
+	var/datum/outfit/O = new outfit
+	O.equip(src)
```
Comment (MrPerson) [inline code/modules/mob/living/carbon/human/inventory.dm]: Maybe also have support for passing an actual datum so later we can like make a datum and save all the clothes to it for reuse later.

**Comment by optimumtact** [comment]
yes

**Comment by WJohn** [comment]
What does this do, exactly? Does it let us set some custom-made select equipment loadouts?

**Comment by optimumtact** [comment]
It makes equipment set changing easier in code

 x = new equipment/type()
 x.equip(themob)

**Comment by AnturK** [comment]
You know what, i'll do antag/misc outfits in another PR. This is finished for now.

**Comment by optimumtact** [comment]
@Cheridan @Razharas merge this it's good!

**Pull request merged**